### PR TITLE
Issue 43 create fuctional test stub

### PR DESCRIPTION
**Pull request by vladislav-kulakovsky**
I don't understand should I add @Before and @After sections?

### CODE REVIEW

File: src/test/java/com/emc/mongoose/storage/driver/pravega/util/docker/MongooseContainer.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+		for (int i = 0; i < env.length; i++) {
+			if ("PATH".equals(env[i])) {
+				env[i] = env[i] + "=" + envMap.get(env[i]) + ":/bin";
+			} else {
+				env[i] = env[i] + "=" + envMap.get(env[i]);
+			}
+		}
+		final CreateContainerResponse container = dockerClient
+				.createContainerCmd(testingImageId)
+				.withName("mongoose")
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/storage/driver/pravega/util/docker/MongooseContainer.java:149]: should be `mongoose-storage-driver-pravega`
https://hub.docker.com/r/emcmongoose/mongoose-storage-driver-pravega/

File: src/test/resources/scenario/test_stub.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                }
+            },
+            "load": {
+                "op": {
+                    "limit": {
+                        "count": TEST_STEP_LIMIT_COUNT
+                    }
+                }
+            },
+            "output": {
```
Comment (akurilov) [inline src/test/resources/scenario/test_stub.js]: please remove the `output` section here as far as we will need the output log files for the validation

**Review by akurilov — COMMENTED** [review]
> I don't understand should I add @before and @after sections?

Nope. The static before/after "class" methods are used for this purpose

File: src/test/java/com/emc/mongoose/storage/driver/pravega/util/docker/MongooseContainer.java (hunk 1 above)
Comment (vladislav-kulakovsky) [reply src/test/java/com/emc/mongoose/storage/driver/pravega/util/docker/MongooseContainer.java:149]: @akurilov I had done it using an [example](https://github.com/emc-mongoose/mongoose-storage-driver-hdfs/blob/e2d977c809022d05c1cde39324fa5d56f1997af0/src/test/java/com/emc/mongoose/storage/driver/hdfs/util/docker/MongooseContainer.java#L152) hdfs driver

Could you explain why in hdfs used name `mongoose` but in pravega I should be used `mongoose-storage-driver-pravega`?

Comment (akurilov) [reply src/test/java/com/emc/mongoose/storage/driver/pravega/util/docker/MongooseContainer.java:149]: I was wrong. We build the testing image right here in this code above, so the name doesn't matter.

File: src/test/resources/scenario/test_stub.js (hunk 2 above)
Comment (vladislav-kulakovsky) [reply src/test/resources/scenario/test_stub.js]: Complete

**Pull request closed without merging**